packages/feedback/package.json

```diff
@@ -42,7 +42,7 @@
     "@availity/api-core": ">=2.0.0",
     "@availity/localstorage-core": "^2.6.1",
     "axios": "^0.18.0",
-    "formik": "^2.0.1-rc.5",
+    "formik": "^2.0.0",
```

Comment:
this was downgraded, any reason was the RC was needed before that might affect this?

Reply:
Good catch. `2.0.0` is deprecated, anyways. i'll change to `^2.0.1`